docker-compose.build.yml

```
@@ -9,17 +9,126 @@ services:
+        SMTP_FROM_ADDRESS: ${SMTP_FROM_ADDRESS:-}
+        SMTP_HOST: ${SMTP_HOST:-}
+        SMTP_SECURE: ${SMTP_SECURE:-true}
+        SMTP_USERNAME: ${SMTP_USERNAME:-}
+        SMTP_PASSWORD: ${SMTP_PASSWORD:-}
+        MEET_JWT_APP_ID: ${MEET_JWT_APP_ID:-}
+        MEET_JWT_APP_SECRET: ${MEET_JWT_APP_SECRET:-}
+        SENTRY_ORG: ${SENTRY_ORG:-}
+        SENTRY_PROJECT: ${SENTRY_PROJECT:-}
+        SENTRY_AUTH_TOKEN: ${SENTRY_AUTH_TOKEN:-}
+        NEXT_PUBLIC_SENTRY_DEBUG: ${NEXT_PUBLIC_SENTRY_DEBUG:-false}
+        NEXT_PUBLIC_SENTRY_DSN: ${NEXT_PUBLIC_SENTRY_DSN:-}
```

YAMLlint (1.37.1)

[error] duplication of key "NEXT_PUBLIC_SENTRY_DSN" in mapping

(key-duplicates)